pkg/sessiontxn/isolation/base.go

```diff
@@ -304,6 +306,14 @@ func (p *baseTxnContextProvider) ActivateTxn() (kv.Transaction, error) {
 		sessVars.SetInTxn(true)
 	}
 
+	// verify start_ts is later than any previous commit_ts in the session
```

Comment:
Is this comment correct? Isnt it checking the LastCommitTS is later than the StartTS instead of the other way around?

Reply:
I think it is correct. It logs fatal if LastCommitTS is later than StartTS.